packages/databricks-vscode/src/bundle/BundleFileSet.ts

```diff
@@ -54,6 +56,22 @@ export class BundleFileSet {
         return Uri.file(rootFile[0]);
     }
 
+    async getSubProjects(): Promise<{relative: Uri; absolute: Uri}[]> {
+        const subProjectRoots = await glob.glob(
+            toGlobPath(this.getAbsolutePath(this.subProjectFilePattern).fsPath),
+            {nocase: process.platform === "win32"}
+        );
+        const delimiter = process.platform === "win32" ? "\\" : "/";
+        return subProjectRoots.map((rootFile) => {
+            const partsWithoutFileName = rootFile.split(delimiter).slice(0, -1);
+            const absolute = Uri.file(partsWithoutFileName.join(delimiter));
```

Comment:
This can be simplified by [path.dirname](https://nodejs.org/api/path.html#pathdirnamepath)